xpcPlugin/Timer.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2013-2018 United States Government as represented by the Administrator of the
+// National Aeronautics and Space Administration. All Rights Reserved.
+
+#ifndef Timer_hpp
+#define Timer_hpp
+
+#include <stdio.h>
+#include <iostream>
+#include <thread>
+#include <chrono>
+#include <functional>
+
+namespace XPC {
+    class Timer
+    {
+        
+        bool running = false;
```

Comment:
`bool` isn't guaranteed to be thread-safe. I think `std::atomic_flag` would be the most appropriate replacement.

Reply:
fixed